CITi-UFPE/CITiMarco
# update(feat/input-scren-2): creating button component

##What I've done
- Created button component

## How to test

1. yarn
2. yarn build
3. yarn start

Go to http://localhost:3000/meetings and see if the button looks like with the mockup

#33
